src/bbs2gh/Commands/MigrateRepo/MigrateRepoCommandArgs.cs

```diff
@@ -92,11 +93,6 @@ public override void Validate(OctoLogger log)
 
     private void ValidateNoGenerateOptions()
     {
-        if (BbsUsername.HasValue() || BbsPassword.HasValue())
```

Comment:
Removing this, I don't think it should be checked for with [this change](https://github.com/github/gh-gei/pull/1057)

Reply:
the check should probably still exist but the error message needs to be changed because this is validating the args for a no generate scenario where the cli doesn't generate the archive and the archive path or url is is provided.
That being said I'd recommend revising the error messages in a separate PR because there is more than just this one, for example the one on line 105 says "SSH or SMB download options can only be provided with --bbs-server-url." but based on the same logic the server url should always be provided.

Reply:
Since we now [require](https://github.com/github/gh-gei/pull/1057) the `bbs-server-url`, we always check for a username and password, even in the non-generate archive case.

`--bbs-server-url` enables the create of the bbsApiFactory, which then does the check for username and password.